wheel/src/run_generator.rs

```diff
@@ -42,52 +42,48 @@ pub struct PySpendBundleConditions {
     pub seconds_absolute: u64,
     // Unsafe Agg Sig conditions (i.e. not tied to the spend generating it)
     #[pyo3(get)]
-    pub agg_sig_unsafe: Vec<(PyObject, PyObject)>,
+    pub agg_sig_unsafe: Vec<(Bytes48, Bytes)>,
     #[pyo3(get)]
     pub cost: u64,
 }
 
-fn node_to_pybytes(py: Python, a: &Allocator, n: NodePtr) -> PyObject {
-    PyBytes::new(py, a.atom(n)).into()
-}
-
-fn convert_spend(py: Python, a: &Allocator, spend: Spend) -> PySpend {
-    let mut agg_sigs = Vec::<(PyObject, PyObject)>::new();
+fn convert_spend(a: &Allocator, spend: Spend) -> PySpend {
+    let mut agg_sigs = Vec::<(Bytes48, Bytes)>::new();
     for (pk, msg) in spend.agg_sig_me {
-        agg_sigs.push((node_to_pybytes(py, a, pk), node_to_pybytes(py, a, msg)));
+        agg_sigs.push((a.atom(pk).into(), a.atom(msg).into()));
     }
-    let mut create_coin = Vec::<(PyObject, u64, PyObject)>::new();
+    let mut create_coin = Vec::<(Bytes32, u64, Option<Bytes>)>::new();
     for c in spend.create_coin {
         create_coin.push((
-            PyBytes::new(py, &c.puzzle_hash).into(),
+            c.puzzle_hash,
             c.amount,
-            node_to_pybytes(py, a, c.hint),
+            if c.hint != a.null() {
+                Some(a.atom(c.hint).into())
+            } else {
+                None
```

Comment:
This change from empty bytes to `None` is also reflected in the test here: https://github.com/Chia-Network/chia_rs/pull/14/files#diff-f584716cd024cb6e7ff78c2fbc0136c01f9742ab26c49f24642262b5b61c68beR84